statsmodels/genmod/families/varfuncs.py

```diff
@@ -45,14 +45,11 @@ def __call__(self, mu):
         mu = np.asarray(mu)
         return np.ones(mu.shape, np.float64)
 
-
     def deriv(self, mu):
         """
         Derivative of the variance function v'(mu)
         """
-        from statsmodels.tools.numdiff import approx_fprime_cs
-        # TODO: diag workaround proplem with numdiff for 1d
-        return np.diag(approx_fprime_cs(mu, self))
+        return np.zeros_like(mu)
```

Comment:
I think we will be better off introducing a separate `Constant` variance class. 

But, AFAIK there is currently no usecase anymore for inheritance from VarianceFunction inside statsmodels.